force-app/main/default/classes/ListDonation_TEST.cls

```diff
@@ -0,0 +1,53 @@
+@IsTest
+public class ListDonation_TEST {
+
+    @IsTest
+    public static void testGetDonationHistory() {
+        Test.startTest();
+		
+		Date myDateTime = Date.newInstance(2020, 2, 2);
+
+        Account mockDefaultAccount = new Account(Name = 'Default Account');
+        insert mockDefaultAccount;
+
+		Contact mockDefaultContact = new Contact(FirstName='Alice', LastName='Patterson', AccountId=mockDefaultAccount.Id);
+		Contact secondaryContact = new Contact(FirstName='Josh', LastName='Patterson', AccountId=mockDefaultAccount.Id);
+		insert mockDefaultContact;
+		insert secondaryContact;
+
+		Opportunity opportunityHardCredit = new Opportunity(Name='HardCredit1', AccountId=mockDefaultAccount.Id, Primary_Contact__c=mockDefaultContact.Id, Amount=400, StageName='Closed Won', CloseDate=myDateTime);
+		Opportunity opportunitySoftCredit = new Opportunity(Name='SoftCredit1', AccountId=mockDefaultAccount.Id, Primary_Contact__c=secondaryContact.Id, Amount=200, StageName='Closed Won', CloseDate=myDateTime);
+		Opportunity opportunityPartialSoftCredit = new Opportunity(Name='PartialSoftCredit1', AccountId=mockDefaultAccount.Id, Primary_Contact__c=secondaryContact.Id, Amount=150, StageName='Closed Won', CloseDate=myDateTime);
+		insert opportunityHardCredit;
+		insert opportunitySoftCredit;
+		insert opportunityPartialSoftCredit;
+
+		
+		OpportunityContactRole oppCntRoleSoftCredit = new OpportunityContactRole(Role='Soft Credit', OpportunityId=opportunitySoftCredit.Id, ContactId=mockDefaultContact.Id, isPrimary=false);
+		OpportunityContactRole oppCntRolePartialSoftCredit = new OpportunityContactRole(Role='Soft Credit', OpportunityId=opportunityPartialSoftCredit.Id, ContactId=mockDefaultContact.Id, isPrimary=false);
+		insert oppCntRoleSoftCredit;
+		insert oppCntRolePartialSoftCredit;
+        
+        Partial_Soft_Credit__c partialSoftCredit = new Partial_Soft_Credit__c
```

Comment:
Some of the formatting here gained a bit of whitespace, can we get this Partial Soft Credit constructor to be reformatted to not be shifted so far to the right?